contrib/upload-kktix-ticket-csv-to-bigquery.py

```diff
@@ -257,131 +392,362 @@ def main():
     return sanitized_df.columns
 
 
-class Test2020Ticket(unittest.TestCase):
+class TestCrossYear(unittest.TestCase):
     """python -m unittest upload-kktix-ticket-csv-to-bigquery.py"""
 
-    CANONICAL_COLUMN_NAMES_2020 = [
-        "ticket_type",
-        "payment_status",
-        "tags",
-        "paid_date",
-        "price",
-        "invoice_policy",
-        "invoiced_company_name",
-        "unified_business_no",
-        "dietary_habit",
-        "years_of_using_python",
-        "area_of_interest",
-        "organization",
-        "job_title",
-        "country_or_region",
-        "departure_from_region",
-        "how_did_you_know_pycon_tw",
-        "have_you_ever_attended_pycon_tw",
-        "know_financial_aid",
-        "gender",
-        "pynight_attendee_numbers",
-        "pynight_attending_or_not",
-        "email_from_sponsor",
-        "email_to_sponsor",
-        "ive_already_read_and_i_accept_the_epidemic_prevention_of_pycon_tw",
-        "ive_already_read_and_i_accept_the_privacy_policy_of_pycon_tw",
-        "email",
-    ]
+    def test_columns_intersection(self):
+        set_extra = {"tags", "attendance_book"}
+
+        set_2020 = set(CANONICAL_COLUMN_NAMES_2020_CORE)
+        set_2019 = set(CANONICAL_COLUMN_NAMES_2019_CORE)
+        set_2018 = set(CANONICAL_COLUMN_NAMES_2018_CORE)
```

Comment:
Why the extra `set()` call? These are already sets. (Same applies for other `set()` calls below.)

Reply:
Good catch. Stupid me. Not working on this pull request with a bug chunk of time block, and forgot to update this part.